pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,127 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc_store {
+    //index-client
+    rpc BatchWrite(WriteBatch) returns (Nothing);
+    rpc QueryPages(IndexQuery) returns (stream ReadBatch);
+
+    //storage-client
+    rpc PutChunks(ChunksData) returns (Nothing);
+    rpc GetChunks(Chunks) returns (ChunksResponse);
+    rpc DeleteChunks(ChunkID) returns (Nothing);
+
+    //table-client
+    rpc ListTables(Nothing) returns (ListTablesResponse);
+    rpc CreateTable(TableDesc) returns (Nothing);
+    rpc DeleteTable(TableName) returns (Nothing);
+    rpc DescribeTable(TableName) returns (DescribeTableResponse);
+    rpc UpdateTable(UpdateTableRequest) returns (Nothing);
+
+    //used by storage-client & index-client
+    rpc Stop(Nothing) returns (Nothing);
+}
+
+message ChunksData {
+    repeated Chunk chunks = 1;
+}
+
+message ChunksResponse {
+    repeated ChunkResponse chunks = 1;
+}
+
+message ChunkResponse {
+    uint32 checksum = 1;
```

Comment:
Why are we returning the checksum here? Further, I checked the Cassandra grpc implementation code and you're just returning the same checksum as the caller?

Reply:
@gouthamve I am returning the same checksum. Shall I drop it from the response?

Reply:
Dropped the checksum here.